setup.json

```diff
@@ -33,7 +33,8 @@
     "ipython~=7.0",
     "jinja2~=2.10",
     "kiwipy[rmq]~=0.5.1",
-    "numpy~=1.17,<1.18",
+    "numpy~=1.17,<1.18; python_version>='3.6'",
+    "numpy>=1.17.0,<1.17.5; python_version=='3.5'",
```

Comment:
I think we should actually revert that change, since I believe that it did not have the desired effect. I'll test that as part of the next iteration.

Reply:
ok